lib/matplotlib/cbook.py

```diff
@@ -2403,12 +2403,16 @@ def pts_to_midstep(x, *args):
     # convert 2D array back to tuple
     return tuple(steps)
 
-STEP_LOOKUP_MAP = {'pre': pts_to_prestep,
+STEP_LOOKUP_MAP = {'default': lambda x, y: (x, y),
+                   'pre': pts_to_prestep,
                    'post': pts_to_poststep,
                    'mid': pts_to_midstep,
                    'step-pre': pts_to_prestep,
                    'step-post': pts_to_poststep,
-                   'step-mid': pts_to_midstep}
+                   'step-mid': pts_to_midstep,
```

Comment:
I would remove the `step-*`  entries.  They are not documented and this is the only place those strings appear in the code base.  I am 98% sure the lack of 's' here was my mistake.